src/Gzero/Core/Handler/Block/Basic.php

```diff
@@ -34,10 +34,10 @@ public function load(Block $block, Lang $lang)
      */
     public function render()
     {
-        return \View::make(
-            'blocks.basic',
-            ['block' => $this->block, 'translations' => $this->block->getTranslations()->first()]
-        )->render();
+        //return \View::make(
+        //    'blocks.basic',
+        //    ['block' => $this->block, 'translations' => $this->block->getTranslations()->first()]
+        //)->render();
```

Comment:
We use it because this is rendered to $block->view property, as you can see [here](https://github.com/GrupaZero/cms/blob/master/src/Gzero/Core/BlockHandler.php#L101). Then when we sent blocks to view, we only need to display rendered block by using $block->view